admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/config/SyncSwaggerConfig.java

```diff
@@ -50,11 +49,11 @@ public OpenAPI openApi() {
 		return api;
     }
 	
-	@Bean
+	/*@Bean
 	public GroupedOpenApi groupedOpenApi() {
```

Comment:
Check by uncommenting this if it works.

Reply:
Uncommented  and imported the valid imports for GroupedOpenApi, working as expected